v1/mobileapps.yaml

```diff
@@ -270,54 +270,3 @@ paths:
                             from_backend.headers) }}'
               body: '{{from_backend.body}}'
       x-monitor: false
-
-  /mobile-text/{title}:
```

Comment:
+1 to what @d00rman said. I made a section about [route usage](https://www.mediawiki.org/wiki/Wikimedia_Apps/Team/RESTBase_services_for_apps#Route_usage) on the MCS page, with some links to some Grafana reports. I remember seeing mobile-text there but cannot find it at the moment. Is it supposed to be at https://grafana.wikimedia.org/dashboard/db/restbase?panelId=15&fullscreen&from=1487062171322&to=1487071346871?